docs/content.zh/docs/connectors/table/dynamodb.md

```diff
@@ -0,0 +1,306 @@
+---
+title: DynamoDB
+weight: 5
+type: docs
+aliases:
+- /zh/dev/table/connectors/dynamodb.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Amazon DynamoDB SQL Connector
+
+{{< label "Sink: Batch" >}}
+{{< label "Sink: Streaming Append & Upsert Mode" >}}
+
+The DynamoDB connector allows for writing data into [Amazon DynamoDB](https://aws.amazon.com/dynamodb).
+
+Dependencies
+------------
+
+{{< sql_download_table "dynamodb" >}}
```

Comment:
@zentol is this the correct reference for SQL download given the connector versioning strategy?

Reply:
No; that still needs some work.
Either we add a new shortcode, _or_ we set the version in `flink/docs/data/sql_connectors.yml`.

Reply:
Setting to `{{< sql_connector_download_table dynamodb 3.0.0 >}}` for now and we can make this work Flink side